### PR TITLE
add build create

### PR DESCRIPTION
**Pull request by wzjgo1**

### CODE REVIEW

File: drone/client.go [hunk 1]
```diff
@@ -229,6 +230,15 @@ func (c *client) Build(owner, name string, num int) (*Build, error) {
 	return out, err
 }
 
+// Manually build.
+func (c *client) BuildCreate(owner, name string, branch string, commit string) (*Build, error) {
+	out := new(Build)
+	uri := fmt.Sprintf(pathBuildCreate, c.addr, owner, name)
```
Comment (tboerger) [inline drone/client.go]: You are anyway adding a query value, instead of creating a new route just reuse pathBuilds

Comment (wzjgo1) [reply drone/client.go]: OK

**Comment by JulienTant** [comment]
Hi, any news on that ?

File: drone/client.go [hunk 2]
```diff
@@ -229,6 +229,15 @@ func (c *client) Build(owner, name string, num int) (*Build, error) {
 	return out, err
 }
 
+// Manually build.
```
Comment (JulienTant) [inline drone/client.go]: From https://golang.org/doc/effective_go.html#commentary :

> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

You should start your comment with "BuildCreate"

File: drone/interface.go [hunk 3]
```diff
@@ -81,6 +81,9 @@ type Client interface {
 	// Build returns a repository build by number.
 	Build(namespace, name string, build int) (*Build, error)
 
+	// Manually build
```
Comment (JulienTant) [inline drone/interface.go]: same remark as before

File: drone/client.go (hunk 2 above)
Comment (wzjgo1) [reply drone/client.go]: thanks

**Comment by abogatikov** [comment]
Are there any plans to merge this PR? @tboerger

**Comment by abogatikov** [comment]
Hi there!
I found that the server endpoint has support for additional parameters, but this implementation doesn't use this. Can we fix that?

**Comment by tboerger** [comment]
Sorry, totally forgot this PR. I have provided a similar one which had been merged today. So this can get closed.

**Comment by tboerger** [comment]
Solved via https://github.com/drone/drone-go/pull/49

**Pull request closed without merging**